CHANGELOG.asciidoc

```diff
@@ -38,6 +38,8 @@ Notes:
 [float]
 ===== Features
 
+* Enable support for redis v4 ({pull}2945[#2945])
```

Comment:
Will need to move this to a new "Unreleased" section now that v3.39.0 is out.

Reply:
✅